pkgs/data/misc/cacert/default.nix

```diff
@@ -17,10 +17,14 @@
 }:
 
 let
-  blocklist = writeText "cacert-blocklist.txt" (lib.concatStringsSep "\n" blacklist);
+  blocklist = writeText "cacert-blocklist.txt" (lib.concatStringsSep "\n" (blacklist ++ [
+    # GLOBALTRUST 2020: Set Distrust After for TLS and S/MIME after 2024/06/30
+    # https://bugzilla.mozilla.org/show_bug.cgi?id=1901080
+    "GLOBALTRUST 2020"
```

Comment:
I don't really know if we should do this; consumers of the certstore via p11kit will pick up distrust-after bit properly anyway (assuming that they're using NSS... which should include Chromium and Firefox at least, anyway).

I'm more thinking of this _now_ in time for the impending Entrust distrust; I don't mind as much for GLOBALTRUST in this case.

Certificate policy is hard.